packages/react/src/components/Storage/FileUploader/FileUploader/FileUploader.tsx

```diff
@@ -63,6 +62,10 @@ export function FileUploader({
     ...overrideDisplayText,
   };
 
+  const allowMultipleFiles =
```

Comment:
Wondering if we can remove this entirely and just infer what we need from `maxFileCount`? Might be easier to do later once the file state lifecycle has been refactored

Reply:
Yeah, I can revisit when I do the file state lifecycle.